test/runner/requirements/units.txt

```diff
@@ -21,3 +21,6 @@ ipaddress
 f5-sdk ; python_version >= '2.7'
 f5-icontrol-rest ; python_version >= '2.7'
 deepdiff
+
+# requirements for HPE OneView specific modules
+hpOneView
```

Comment:
I guess you don't need to create a separare block if you it in the alphabetical order... Beside that, the hpOneView python_version requirement is >= '2.7' isn't it?